tests/lxc/install-deps/images_archlinux

```diff
@@ -26,7 +26,7 @@ pip3 install pytest requests pyyaml
 n=0
 until [ $n -ge 7 ]
 do
-  sudo snap install core18 && break  # substitute your command here
+  sudo snap install core20 && break  # substitute your command here
```

Comment:
why this version bump here and in other places? Seems unrelated to the cleanup.

Reply:
this is actually a leftover from our upgrade core18 -> core20. the `core18` installed here has no effect